charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -92,5 +92,8 @@ spec:
       volumeSnapshotLocation: []
     metrics:
       serviceMonitor:
+        {{- if .Values.monitoring.prometheus.enabled }}
```

Comment:
The `if` is not needed, we can just always just enable the serviceMonitor, as the chart checks for the existence of the CRD

Reply:
removed